modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,309 @@
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 01-master-interface
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<encoding_config>
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/sysconfig/network-scripts/ifcfg-<connection_name>
+    systemd: {}
+----
++
+--
+where:
+
+`<encoding_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
+`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
+--
+
+** If your hardware MTU is specified with DHCP, update your DHCP configuration such as with the following dnsmasq configuration:
++
+[source,text]
+----
+dhcp-option-force=26,<mtu>
+----
++
+--
+where:
+
+`<mtu>`:: Specifies the hardware MTU for the DHCP server to advertise.
+--
+
+** If your hardware MTU is specified with a kernel command line with PXE, update that configuration accordingly.
+
+. To begin the MTU migration, specify the migration configuration by entering the following command. The Machine Config Operator performs a rolling reboot of the nodes in the cluster in preparation for the MTU change.
++
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type=merge --patch \
+  '{"spec": { "migration": { "mtu": { "network": { "from": <overlay_from>, "to": <overlay_to> } , "machine": { "to" : <machine_to> } } } } }'
+----
++
+--
+where:
+
+`<overlay_from>`:: Specifies the current cluster network MTU value.
+`<overlay_to>`:: Specifies the target MTU for the cluster network. This value is set relative to the value for `<machine_to>` and for OVN-Kubernetes must be `100` less and for OpenShift SDN must be `50` less.
+`<machine_to>`:: Specifies the MTU for the primary network interface on the underlying host network.
+--
++
+.Example that increases the cluster MTU
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type=merge --patch \
+  '{"spec": { "migration": { "mtu": { "network": { "from": 1400, "to": 9000 } , "machine": { "to" : 9100} } } } }'
+----
+
+. To confirm that all of the nodes in your cluster were rebooted successfully, enter the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+. Update the underlying network interface MTU value:
+
+** If are specifying the new MTU with a NetworkManager connection configuration, enter the following command. The MachineConfig Operator automatically performs a rolling reboot of the nodes in your cluster.
++
+[source,terminal]
+----
+$ for manifest in machine-config-control-plane machine-config-worker; do
+    oc create -f $manifest.yaml
+  done
+----
+
+** If are specifying the new MTU with a DHCP server option or a kernel command line and PXE, reboot each node manually.
```

Comment:
This specific reboot is not strictly necessary, as the reboot on the next step would acquire the changes.